Jenkinsfile

```diff
@@ -38,6 +39,20 @@ pipeline {
                                 disableDeferredWipeout: true,
                                 notFailBuild: true)
 		}
+
+        success {
+            gitShortCommit = sh(returnStdout: true, script: 'git rev-parse --short HEAD').trim()
+            script {
+                slackMessage = "ope.fda.gov UI production (<https://github.com/FDA/open.fda.gov/commits/${gitShortCommit}|${gitShortCommit}>) deployed to production.)"
```

Comment:
this environment can be parameterized if we want to do that way (eg. test, dev, etc.)